# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1251101212 -7200
#      Mon Aug 24 10:06:52 2009 +0200
# Node ID 163e205600a4283ba5d41b165fe93dced01c6823
# Parent  8b3cee6166b8c34f00f7bc530dd939b18fe2c86d
fix typo in acx_blas_f77_func.m4

diff --git a/ChangeLog b/ChangeLog
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,8 +1,12 @@
+2009-08-24  Jaroslav Hajek  <highegg@gmail.com>
+
+	* acx_blas_f77_func.m4: Fix typo.
+
 2009-08-19  Jaroslav Hajek  <highegg@gmail.com>
 
 	* NEWS: Update.
 
 2009-08-18  John W. Eaton  <jwe@octave.org>
 
 	* aclocal.m4 (OCTAVE_HDF5_HAS_REQUIRED_API): If test fails, set
 	octave_hdf5_ok to no instead of setting HAVE_HDF5 to false.
diff --git a/acx_blas_f77_func.m4 b/acx_blas_f77_func.m4
--- a/acx_blas_f77_func.m4
+++ b/acx_blas_f77_func.m4
@@ -91,17 +91,17 @@ elif test x"$acx_blas_ok" = xyes; then
       a(1) = 1e0
       a(2) = -2e0
       i = isamax(2,a,1)
       if (i.ne.2) stop 1
       ]]),[acx_blas_isamax_fcall_ok=yes],
 	[acx_blas_isamax_fcall_ok=no])
 	AC_MSG_RESULT([$acx_blas_isamax_fcall_ok])
 # SDOT check (REAL return values)
-	AC_MSG_CHECKING([whether DDOT is called correctly from Fortran])
+	AC_MSG_CHECKING([whether SDOT is called correctly from Fortran])
 	AC_RUN_IFELSE(AC_LANG_PROGRAM(,[[
       real sdot,a(1),b(1),w
       external sdot
       a(1) = 1e0
       b(1) = 2e0
       w = sdot(1,a,1,b,1)
       if (w .ne. a(1)*b(1)) stop 1
       ]]),[acx_blas_sdot_fcall_ok=yes],
